frontend/src/pages/mock/login.page.tsx

```diff
@@ -0,0 +1,124 @@
+import { NextPage } from "next";
+import { useRouter } from "next/router";
+import { FormEventHandler, useEffect, useState } from "react";
+import styles from "./mockSession.module.scss";
+import { getRuntimeConfig } from "../../config";
+import { apiFetch } from "../../hooks/api/api";
+import { UsersData } from "../../hooks/api/user";
+
+type UsedToken = {
+  token: string;
+  user: string;
+  lastUsed: number;
+};
+
+const MockLogin: NextPage = () => {
+  const router = useRouter();
+  const [usedTokens, setUsedTokens] = useState<UsedToken[]>([]);
+  const [token, setToken] = useState("");
+
+  useEffect(() => {
+    const usedTokensString = localStorage.getItem("usedTokens") ?? "[]";
+    setUsedTokens(JSON.parse(usedTokensString).sort(byUseDate));
+  }, []);
+
+  useEffect(() => {
+    if (process.env.NODE_ENV !== "development") {
+      router.replace("/404");
```

Comment:
It would, but on production that is handled by Django, so I think that's outside of the scope of this PR... (This page is only used when running a local dev server with `npm run dev`.) I'm not sure why this code is in there though - probably a leftover from when I was struggling to get authn working in the first place, so I removed it: 8ba8b91bd6f830e8a6e594827b26788dc2f57483.